activerecord/test/cases/locking_test.rb

```diff
@@ -415,14 +415,16 @@ def test_counter_cache_with_touch_and_lock_version
     assert_equal 1, car.lock_version
 
     previously_car_updated_at = car.updated_at
-    car.wheels.first.update(size: 42)
```

Comment:
The failure is due to datetime rounding, but daf15f58b943d85d8fb726590ae94f77ca0a5d5f didn't fix that.
that is caused by this line, so we need to address this line only.

Reply:
Done!